src/client/components/ModalWithFeedback/ModalWithFeedback.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import { Button } from '../Button/Button';
+import PropTypes from 'prop-types';
+import './ModalWithFeedback.css';
+import Input from '../Input/Input';
+import { EventCardText } from '../EventCardText/EventCardText';
+import Modal from '../Modal/Modal';
+
+function ModalWithFeedback(props) {
+  const [text, textName] = useState('');
+
+  const handelSubmit = (e) => {
+    e.preventDefault();
```

Comment:
Now that you handled click, you need to call a handler in props called `onSubmit` and pass `text` as parameter.